tutorial/ndarrays.md

```diff
@@ -326,3 +310,19 @@ Some examples:
 
         end program prog_ex
         ```
+
+## Numpy [ndarray](https://numpy.org/doc/stable/reference/generated/numpy.ndarray.html) functions/properties progress in Pyccel ##
+
+-   Supported [types](https://numpy.org/devdocs/user/basics.types.html):
+
+    bool, int, int8, int16, int32, int64, float, float32, float64, complex64 and complex128. They can be used as cast functions too.
+
+    Note: np.bool, np.int and np.float are just aliases to the Python native types, and are considered as a deprecated way to work with Python built-in types in NumPy.
+
+-   Properties:
+
+    -   real, imag, shape, amax, amin
+
+-   Methods:
+
+    - sum
```

Comment:
As reported by Codacy, The indentation on line 328 does not match the indentation on line 324